model-optimizer/requirements_tf.txt

```diff
@@ -1,7 +1,6 @@
-# TensorFlow 1.x and 2.x are incompatible, use separate virtual environments for each version
```

Comment:
The deprecation process works differently. We first add a warning message that something is deprecated but still allow user to use it and then in next (or even the next after the next) release we remove possibility to use the deprecated feature. Here we completely remove the ability to install TF1 from requirements file without notice period. I believe that some customers may have installation scripts which configure the environment and use these files. May be we should consider more graceful transition?

CC @jane-intel , @andrei-kochin , @rkazants .

Reply:
@undef-nnov There was a deprecation notice in the Release notes before even if it was not stated explicitly in the warning from code. There is a still possibility to use TF1 environment but starting from this release it will no longer be recommended and supported by OpenVINO. TF1 models should be inferred through TF2 compatibility mode.